src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -293,130 +298,129 @@ const MapView: React.FC<MapViewProps> = ({
 
-        // Asks user if they want to show their current location
-        if (navigator.geolocation) {
-          navigator.geolocation.getCurrentPosition(
-            (pos) => {
-              const me = new google.maps.LatLng(
-                pos.coords.latitude,
-                pos.coords.longitude,
+            // Sets the information to display in the popup
+            setActiveTreeInfo({
+              id: siteId,
+              commonName: eventFeature.getProperty('commonName'),
+              address: eventFeature.getProperty('address'),
+            });
+            // Popup appears at the site
+            eventFeature
+              .getGeometry()
+              .forEachLatLng((latLng: google.maps.LatLng) =>
+                popup.popAtPosition(latLng),
               );
-              // eslint-disable-next-line
-              const userLocation = new google.maps.Marker({
-                position: me,
-                map,
-                clickable: false,
-                icon: {
-                  path: google.maps.SymbolPath.CIRCLE,
-                  scale: 5,
-                  fillColor: 'blue',
-                  fillOpacity: 1,
-                  strokeWeight: 0,
-                },
-              });
-            },
-            () => {
-              message
-                .info(
-                  'Enable access to your location to display where you are on the map.',
-                  5,
-                )
-                .then();
-            },
-          );
-        }
+          });
 
-        // Shows or hides layers based on the zoom level
-        function handleZoomChange() {
-          google.maps.event.addListener(map, 'zoom_changed', () => {
-            const zoomLevel = map.getZoom();
-            let zoomedIn = false;
+          // Initially hidden while the neighborhoods are shown
+          setSitesStyle(sitesLayer, 0, false);
+
+          // Asks user if they want to show their current location
+          if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(
+              (pos) => {
+                const me = new google.maps.LatLng(
+                  pos.coords.latitude,
+                  pos.coords.longitude,
+                );
+                // eslint-disable-next-line
+                const userLocation = new google.maps.Marker({
+                  position: me,
+                  map,
+                  clickable: false,
+                  icon: {
+                    path: google.maps.SymbolPath.CIRCLE,
+                    scale: 5,
+                    fillColor: 'blue',
+                    fillOpacity: 1,
+                    strokeWeight: 0,
+                  },
+                });
+              },
+              () => {
+                message
+                  .info(
+                    'Enable access to your location to display where you are on the map.',
+                    5,
+                  )
+                  .then();
```

Comment:
Does this then() do anything? 🤔

Reply:
Not really, it just gets rid of the warning that the promise from the message is ignored